apps/mobile/app/signin.tsx

```diff
@@ -121,7 +121,7 @@ export default function Signin() {
                   placeholder="Email"
                   keyboardType="email-address"
                   autoCapitalize="none"
-                  value={usernameFormData.email}
+                  value={usernameFormData.email.trim()}
                   onChangeText={(e) =>
                     setUserNameFormData((s) => ({ ...s, email: e }))
```

Comment:
This needs to happen in the setter otherwise it won't make the desired effect

```suggestion
                  value={usernameFormData.email}
                  onChangeText={(e) =>
                    setUserNameFormData((s) => ({ ...s, email: e.trim() }))
```